partcad/src/partcad/runtime_python_conda.py

```
@@ -80,19 +82,24 @@ def once_conda_locked(self):
+                        span.set_attribute("cmd", " ".join(args))
+                        # Install new conda environment with the preferred Python version
+                        p = subprocess.Popen(
+                            args,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            shell=False,
+                            encoding="utf-8",
+                        )
+                        _, stderr = p.communicate()
                     if not stderr is None and stderr.strip() != "":
                         pc_logging.error("conda env install error: %s" % stderr)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)